make-dist.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+set -eu
+
+main() {
+  local target=''
+  # "Darwin x86_64" becomes "darwin-x86_64".
+  target="$(uname -sm | tr ' ' '-' | tr '[:upper:]' '[:lower:]')"
```

Comment:
If we're willing to go this far just because we want a lowercase directory name...
```suggestion
  target="$(uname -sm | tr '[:upper:] ' '[:lower:]-')"
```

I'm still not a fan, and I'd rather just use `Darwin-x86_64` and similar as directory names, so we could just do
`target=$(uname -s)-$(uname -m)` which is far easier to glance at.

Reply:
Done, kept uppercase Darwin per suggestion.